.github/workflows/main.yml

```diff
@@ -10,67 +10,83 @@ jobs:
 
     steps:
     # checkout repo
-    - uses: actions/checkout@v4
+    - name: Checkout
+      uses: actions/checkout@v4
 
-    # setup multi-arch build support
-    - name: Install Dependencies
-      run: |
-        sudo apt-get update
-        sudo apt-get --yes --no-install-recommends install binfmt-support qemu-user-static
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
 
-    # get branch / tag name
     - name: Get Branch / Tag Name
       id: get_branch
       run: |
         export BRANCH_NAME=$(if [[ ${GITHUB_REF} =~ "refs/tags/" ]]; then echo ${GITHUB_REF/refs\/tags\//}; else echo ${GITHUB_REF/refs\/heads\//}; fi)
         echo $BRANCH_NAME
         echo "BRANCH_NAME=${BRANCH_NAME}" >> $GITHUB_OUTPUT
 
-    # generate the image tag
     - name: Get Image Tag
       id: get_tag
       run: |
-        export TARGET_IMAGE_TAG=$(if [ "${{ steps.get_branch.outputs.NAME }}" = "latest" ]; then echo "latest"; else echo "${{ steps.get_branch.outputs.BRANCH_NAME }}"; fi)
+        export TARGET_IMAGE_TAG=$(if [[ "${{ steps.get_branch.outputs.NAME }}" =~ (latest|master|main) ]]; then echo "latest"; else echo "${{ steps.get_branch.outputs.BRANCH_NAME }}"; fi)
         echo $TARGET_IMAGE_TAG
         echo "TARGET_IMAGE_TAG=${TARGET_IMAGE_TAG}" >> $GITHUB_OUTPUT
 
-    # generate the alternative image tag
     - name: Get Alternate Tag
```

Comment:
What is the alternate tag being used for? There is only Ubuntu releases at the moment.